packages/playground/react-emotion/vite.config.ts

```diff
@@ -1,6 +1,7 @@
 import react from '@vitejs/plugin-react'
+import type { UserConfig } from 'vite'
 
-const config: import('vite').UserConfig = {
+const config: UserConfig = {
```

Comment:
Why was the `import('vite').UserConfig` used here before?

Reply:
To prevent exactly what is named in additional context 🙂 
This is now working with type imports from TS 4.4

Reply:
Awesome, thanks for the details!